### ISSUE TITLE
Suggestion: Remove slide right animation changing menu items 

### ISSUE THREAD

**Issue by swedishfrenchpress** [Ride-The-Lightning/RTL-Design#61]
In 0.10.1 I noticed a new slide right animation that appears when the user navigates between menu options. I think the use of these animations might violate some accessibility best practices. I would recommend removing these animations in the next update. 

> It’s important to note that excessive use of motion and animation is an accessibility issue: animations with hard cuts between colors or flashing can trigger seizures in epileptic users.  Parallax, carousel-forwarding animations, and scroll jacking can make users with vestibular disorders dizzy or nauseated and trigger migraines.  Restraint is important, and you should respect those users who have set their browser or device to “reduce motion” by removing 

source: https://www.nngroup.com/articles/animation-duration/

**Comment by ShahanaFarooqui** [comment]
@swedishfrenchpress Can we close this issue against PR [#642](https://github.com/Ride-The-Lightning/RTL/issues/642) or it needs any other fix?

**Comment by swedishfrenchpress** [comment]
@ShahanaFarooqui yes! I'll close.
